app/views/interactive_pages/_edit_section.html.haml

```diff
@@ -0,0 +1,48 @@
+:ruby
+  if section_name == 'HEADER_BLOCK'
```

Comment:
Probably `'HEADER_BLOCK'` should be replaced with `InteractivePage::HEADER_BLOCK`